addons/medical_treatment/functions/fnc_treatmentFailure.sqf

```diff
@@ -23,7 +23,11 @@ _args params ["_medic", "_patient", "_bodyPart", "_classname", "_itemUser", "_us
 
 // Return used item to user (if used)
 if (!isNull _itemUser) then {
-    [_itemUser, _usedItem] call EFUNC(common,addToInventory);
+    if !(((_usedItem call BIS_fnc_itemType) select 0) isEqualTo "Magazine") then {
+        [_itemUser, _usedItem] call EFUNC(common,addToInventory);
+    } else {
+        [_itemUser, _usedItem, 1] call EFUNC(common,adjustMagazineAmmo);
+    };
```

Comment:
I never understood why climb up the tree backwards (read: ass 1st) like this, just do a `true` check and deal with `false` in the `else` part like would be logical. A performance gain should be non-existent.

Reply:
Just use `isNotEqualTo` or johnb's suggestion above if that's the case, no need to switch things around